### DISCUSSION TITLE
Encrypted files in FuzzTests

### DISCUSSION THREAD

**Discussion by eswdd** [klauspost/compress#791] [Q&A] [answered]
Hi,

My company doesn't allow us to use Go packages that contain encrypted files without explicit allowlisting once we know the contents of the files. The recent addition of CICD Fuzz testing (#763) appears to contine 2 encrypted files within `zip/testdata/FuzzReader-raw.zip` (according to some heuristics applied on the boundary):
* `14.zip`
* `19e22ee834f02d145ae7072020443bc7ff06a965-2`

Whilst I appreciate the contents of the files are not relevant for the tests, I am required to verify the contents before I can bring the package in. The first appears to be a password protected zip, for which I cannot find the password anywhere, the second I have no idea.

Would it be possible to provide the provenance of these files so that can I verify the contents?

Many thanks
Simon

**Comment by klauspost** [comment, marked as answer]
Both are from https://github.com/dvyukov/go-fuzz-corpus/tree/master/zip/corpus

I suspect at least one is automatically generated by the fuzzer.

It do not know the content of these files - and likely they don't even decrypt to valid data.

**Reply by eswdd** [reply]
Thank you, I'll try and ask in `go-fuzz` but it looks like this might rapidly hit a dead end.

Apologies for not raising as a discussion, I never even noticed the tab.
